content/en/docs/ops/common-problems/network-issues/index.md

```diff
@@ -223,6 +223,79 @@ server {
 }
 {{< /text >}}
 
+## Virtual service with fault injection and retry/timeout policies not working as expected
```

Comment:
Why is this section specifically here in the doc? Maybe better to put new section at the end of the document.

Reply:
Uh, idk tbh, I guess it just "felt right" because every bullet point was about routing while the last one was about TLS and had so many subsections. I'll just move it to the end then. 👍